generate/beat/{beat}/NOTICE

```diff
@@ -0,0 +1,5 @@
+{beat}
+Copyright 2017 {full_name}
```

Comment:
That is even nicer. My change would only set the correct date on the first creation.